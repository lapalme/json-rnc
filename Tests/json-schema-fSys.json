{
    "/": {
        "storage": {
            "type": "disk",
            "device": "/dev/sda1"
        },
        "fstype": "btrfs",
        "readonly": true
    },
    "/var": {
        "storage": {
            "type": "disk",
            "label": "8f3ba6f4-5c70-46ec-83af-0d5434953e5f"
        },
        "fstype": "ext4",
        "options": [ "nosuid" ]
    },
    "/tmp": {
        "storage": {
            "type": "tmpfs",
            "sizeInMB": 64
        }
    },
    "/var/www": {
        "storage": {
            "type": "nfs",
            "server": "my.nfs.server",
            "remotePath": "/exports/mypath"
        }
    }
}
{"/var":{
    "storage":{"type":"disk","device":"/dev/abc/def"},
    "readonly":true,
    "options":["suid","no-web"]
}}