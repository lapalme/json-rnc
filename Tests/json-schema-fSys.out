The 2 objects are valid
